docs/user/tutorial-asgi.rst

```diff
@@ -31,43 +31,41 @@ WSGI tutorial::
       ├── __init__.py
       └── app.py
 
-.. note::
-   Installing `virtualenv <https://docs.python-guide.org/dev/virtualenvs/>`_ is
-   not needed for recent Python 3.x versions. We can simply create a
-   *virtualenv* using the ``venv`` module from the standard library,
-   for instance::
+Installing `virtualenv <https://virtualenv.pypa.io/>`_ is not usually needed
+for recent Python 3.x versions. We can simply create a *virtualenv* using the
+``venv`` module from the standard library::
 
-     $ python3.8 -m venv .venv
-     $ source .venv/bin/activate
+  $ mkdir asgilook
+  $ python3.8 -m venv asgilook/.venv
```

Comment:
Hm, when I was writing this tutorial, this probably made more sense since 3.5 was still around.
Maybe we could just write `python3` now, and reiterate that 3.6+ is needed for ASGI?

Reply:
I'll update this. Let me know how it looks.